### DISCUSSION TITLE
Duda con abogabot y trabajo  en equipo

### DISCUSSION THREAD

**Discussion by JoseFGam** [Launch-X-Latam/MisionFrontEnd#189] [Q&A]
Buena noche disculpe, el proyecto abogabot debe entenderse como ¿una sola página con solo la sección abogabot o una pagina con varias secciones y una sección para abogabot?, también tengo la inquietud de saber si ¿puedo trabajar en conjunto con otro compañero? ya que vivimos juntos, somos familiares y nos retro-alimentamos en la creación de la buyer persona, publico objetivo y nuestros trabajos o nuestro avance es muy parecido, agradezco de antemano la respuesta.

**Comment by maxkario** [comment]
Con respecto a trabajar en conjunto, puede que si, siempre y cuando presenten sus ideas por separadas. 
Y con respecto a AbogaBot, tienes que verlo como si tu fueras el desarrollador web al que una firma de abogados esta solicitando tus servicios para realizar un sitio web destinado a atender a clientes que necesiten asesoria legal.

**Reply by rpajares** [reply]
Hola donde esta esa tarea? cuales son las pautas?

**Reply by snowingt** [reply]
en donde dice intro  ahi esta esa
